Formula/netdata.rb

```diff
@@ -0,0 +1,64 @@
+class Netdata < Formula
+  desc "Get control of your servers. Simple. Effective. Awesome!"
+  homepage "https://my-netdata.io/"
+  url "https://github.com/firehol/netdata/releases/download/v1.9.0/netdata-1.9.0.tar.bz2"
+  sha256 "542b4799aed1ee03b0a0dbd00fae988a788622c431382e6429362fbbb4f0f017"
+
+  depends_on "ossp-uuid"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :run
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}",
+                          "--sbindir=#{bin}"
```

Comment:
We don't usually relocate `sbin` stuff to `bin` unless there's some very compelling reason to do so.